### ISSUE TITLE
[Feature]: API to generate safetensor file

### ISSUE THREAD

**Issue by Vykq** [Gourieff/sd-webui-reactor#235]
### Feature description

It would be great to generate safetensor file with my desired name thought api

**Comment by vinch00** [comment]
+1

**Comment by indusguys** [comment]
I take it you mean to generate a text file with the name of the safetensor file used or as a script to the top of the image? It really needs that feature if you're trialling several safetensors. There's a script for loras, to do that. I've been hoping this would surface for a while.

**Comment by vinch00** [comment]
No, I believe he means to have a feature to generate the face model safetensors file through an API. This could then be used in the future in the regular API by referencing the filename.

cc: @Gourieff -- would be great to have this!
